readme.md

```diff
@@ -81,7 +82,8 @@ Your Rails database has two tables. `students` has the columns `id` and `name`,
 Use ActiveRecord to create a new `pbj` sandwich and make it belong to the student named Geraldo.
 
 ```rb
-# Your answer...
+var Geraldo = Student.create(name: "Geraldo")
+Geraldo.sandwiches.create(type: "pbj", student_id: 1)
 
 ```
 
```

Comment:
No need to specify `student_id: 1`. We know this sandwich will belong to Geraldo because `.create` is preceded by `Geraldo.sandwiches`.